src/main/java/com/codingame/core/PlayerTimeoutException.java

```diff
@@ -0,0 +1,16 @@
+package com.codingame.core;
+
+public class PlayerTimeoutException extends Exception {
+
+    private static final long serialVersionUID = -8569665968534367562L;
+
+    private AbstractPlayer player;
+
+    PlayerTimeoutException(AbstractPlayer player) {
+        this.player = player;
+    }
+
+    public AbstractPlayer getPlayer() {
+        return player;
+    }
```

Comment:
What will you do with the `player`? Don't you have to override the `toString` or the `getMessage` methods?